app/services/validation_beta_service.rb

```diff
@@ -22,6 +22,14 @@ def invite_schools(array_of_urns:, participant_research:, coordinator_research:)
     end
   end
 
+  def notify_induction_coordinators_about_validation
+    School.eligible.find_each do |school|
+      if added_participants_and_not_in_beta(school) || chosen_programme_but_not_added_participants(school)
+        send_validation_notification(school)
```

Comment:
Are we ok with SITs at multiple schools getting lots of notifications?

Reply:
Fixed this now